packages/contracts/deploy/30_plugins/10_plugin-repos/01_create_address_list_voting_repo_conclude.ts

```diff
@@ -19,21 +19,20 @@ const func: DeployFunction = async function (hre: EHRE) {
     PluginRepoFactoryDeployment.address,
     deployer
   );
-  const managingDaoAddress = await getContractAddress('DAO', hre);
 
-  const pluginRepoBase = await pluginRepoFactory.pluginRepoBase();
+  //const managingDaoAddress = await getContractAddress('DAO', hre);
```

Comment:
remove unused code

Reply:
Fixed in 43bfd49e52eb7cfc4b191a3efe45bbe5c4b27d3a